x/pki/x509/x509.go

```diff
@@ -22,7 +22,7 @@ import (
 	"fmt"
 	"strings"
 
-	"github.com/zigbee-alliance/distributed-compliance-ledger/x/pki/types"
+	pkitypes "github.com/zigbee-alliance/distributed-compliance-ledger/types/pki"
```

Comment:
I think it would be better to move this x509 file to "github.com/zigbee-alliance/distributed-compliance-ledger/utils" and keep "x/pki/types" and "x/pki/errors" in "x/pki". We may do it in a separate PR later.